website/docs/quickstarts/duckdb-core-qs.md

```diff
@@ -0,0 +1,144 @@
+- Change the directory to the Jaffle Shop.
+- Create a Python virtual environment.
+- Install and update pip.
+- Install dbt Core and dependencies.
+- Install DuckDB and dependencies.
+- Build the Jaffle Shop project in dbt Core.
+- Generate (`dbt docs generate`) and serve (`dbt docs serve`) dbt docs to your browser (localhost).
+
+For the complete list of installed dependencies, refer to the [requirements.txt](https://github.com/dbt-labs/jaffle_shop_duckdb/blob/duckdb/requirements.txt) file in the jaffle_shop_duckdb GitHub repo. 
+
+<Tabs>
+<TabItem value="Bash" label="bash/zsh (Mac)">
+
+```shell
+git clone https://github.com/dbt-labs/jaffle_shop_duckdb.git
+cd jaffle_shop_duckdb
+python3 -m venv venv
+source venv/bin/activate
+python3 -m pip install --upgrade pip
+python3 -m pip install -r requirements.txt
+source venv/bin/activate
+dbt build
+dbt docs generate
+dbt docs serve
+```
+
+</TabItem>
+<TabItem value="csh" label="csh/tcsh">
+
+```shell
+git clone https://github.com/dbt-labs/jaffle_shop_duckdb.git
+cd jaffle_shop_duckdb
+python3 -m venv venv
+source venv/bin/activate.csh
+python3 -m pip install --upgrade pip
+python3 -m pip install -r requirements.txt
+source venv/bin/activate.csh
+dbt build
+dbt docs generate
+dbt docs serve
+```
+
+</TabItem>
+<TabItem value="powershell" label="Windows Powershell">
+
+```shell
+git clone https://github.com/dbt-labs/jaffle_shop_duckdb.git
+cd jaffle_shop_duckdb
+python -m venv venv
+venv\Scripts\Activate.ps1
+python -m pip install --upgrade pip
+python -m pip install -r requirements.txt
+venv\Scripts\Activate.ps1
+dbt build
+dbt docs generate
+dbt docs serve
+```
+
+</TabItem>
+<TabItem value="codespace" label="GitHub Codespaces">
+
+#### Steps:
+
+1. Ensure you have [Codespaces](https://github.com/features/codespaces) enabled for your GitHub organization or turned on as a beta feature if you're an individual user.
+2. Open the [jaffle_shop_duckdb](https://github.com/dbt-labs/jaffle_shop_duckdb) repo in your browser.
+3. Click the green **Code** button near the top right of the repo's homepage (you may already be on it).
+4. Rather than cloning the repo like you normally would, click the **Codespaces** tab of the pop-out and then click **Create codespace on `duckdb`**.
+<div style={{maxWidth: '400px'}}>
+    <Lightbox src="/img/codespace-quickstart/open_in_codespaces.png" title="Open in Codespaces" />
+    </div>
+
+5. Wait for Codespaces to boot (this may take a few minutes).
+6. Select whether you'd like to use the Web IDE or open the codespace in your local environment.
+7. When the Codespaces opens, a **Task** pane will show up and call `dbt build` to show you how it's done.
+8. (Optional) Install the recommended extensions (like **dbt Power User extension**).
+9. Open up a new terminal and type: `dbt build`
+
+</TabItem>
+</Tabs>
+
+## Query the data
+
+The Jaffle Shop project seeds the data set to DuckDB, which can then be manually queried from the duckcli tool. 
```

Comment:
```suggestion
The Jaffle Shop project seeds the data set to DuckDB, which can then be manually queried from the DuckCLI tool. 
```

since you're referring to the product name (good point!), sugg match its casing like this:

![Screenshot 2023-10-19 at 2 15 08 PM](https://github.com/dbt-labs/docs.getdbt.com/assets/107218380/1be8ff39-e5dc-4d9e-b504-3c1a233dd1c3)